src/locale/locales/de/messages.po

```diff
@@ -830,88 +813,85 @@ msgstr "Feed offline"
 msgid "Feed Preferences"
 msgstr "Feed-Einstellungen"
 
-#: src/view/shell/desktop/RightNav.tsx:65
-#: src/view/shell/Drawer.tsx:311
+#: src/view/shell/desktop/RightNav.tsx:65 src/view/shell/Drawer.tsx:311
 msgid "Feedback"
-msgstr "Rückmeldung"
+msgstr "Feedback"
 
-#: src/view/screens/Feeds.tsx:475
-#: src/view/screens/Profile.tsx:165
+#: src/view/screens/Feeds.tsx:475 src/view/screens/Profile.tsx:165
 #: src/view/shell/bottom-bar/BottomBar.tsx:181
-#: src/view/shell/desktop/LeftNav.tsx:340
-#: src/view/shell/Drawer.tsx:474
+#: src/view/shell/desktop/LeftNav.tsx:340 src/view/shell/Drawer.tsx:474
 #: src/view/shell/Drawer.tsx:475
 msgid "Feeds"
 msgstr "Feeds"
 
 #: src/view/com/auth/onboarding/RecommendedFeeds.tsx:57
 msgid "Feeds are created by users to curate content. Choose some feeds that you find interesting."
-msgstr "Feeds werden von Nutzern erstellt, um Inhalte zu kuratieren. Wählen Sie einige Feeds aus, die Sie interessant finden."
+msgstr "Feeds werden von Nutzern erstellt, um Inhalte zu kuratieren. Wähle einige Feeds aus, die du interessant findest."
 
 #: src/view/screens/SavedFeeds.tsx:156
 msgid "Feeds are custom algorithms that users build with a little coding expertise. <0/> for more information."
-msgstr "Feeds sind benutzerdefinierte Algorithmen, die von den Nutzern mit ein wenig Programmierkenntnissen erstellt werden. <0/> für weitere Informationen."
+msgstr "Feeds sind benutzerdefinierte Algorithmen, die Nutzer mit ein wenig Programmierkenntnisse erstellen. <0/> für mehr Informationen."
 
 #: src/view/screens/Search/Search.tsx:414
 msgid "Find users on Bluesky"
-msgstr "Benutzer auf Bluesky finden"
+msgstr "Nutzer auf Bluesky finden"
 
 #: src/view/screens/Search/Search.tsx:412
 msgid "Find users with the search tool on the right"
-msgstr "Benutzer mit der Suchfunktion auf der rechten Seite finden"
+msgstr "Finde Nutzer mit der Suchfunktion auf der rechten Seite"
 
 #: src/view/com/auth/onboarding/RecommendedFollowsItem.tsx:150
 msgid "Finding similar accounts..."
 msgstr "Suche nach ähnlichen Konten..."
 
 #: src/view/screens/PreferencesHomeFeed.tsx:102
 msgid "Fine-tune the content you see on your home screen."
-msgstr "Optimieren Sie die Inhalte, die Sie auf Ihrem Startbildschirm sehen."
+msgstr ""
 
 #: src/view/screens/PreferencesThreads.tsx:60
 msgid "Fine-tune the discussion threads."
-msgstr "Feinabstimmung der Diskussionsstränge."
+msgstr ""
```

Comment:
`"Einstellungen für Diskussionsverläufe"`

Reply:
A consistent word should be chosen (here and above) for "threads": Diskussionsverläufe/Diskussionsstränge/Beitragsstränge

Reply:
"Strang/Stränge" is unusual (at least in de_DE), 'Verlauf/Verläufe' much better.

Reply:
"Verlauf/Verläufe" sounds best to me.

Reply:
Or maybe just "Diskussionen".

Reply:
I think this is one of those cases where simply keeping the English `Thread(s)` might be the best approach. I don't think there's any generally accepted/used word for this. At least none I could think of right now (apart from stuff typically used in forums, which doesn't really fit here).